modelskill/timeseries/_point.py

```diff
@@ -151,5 +158,14 @@ def _parse_point_input(
         ds[aux_item].attrs["kind"] = "aux"
 
     ds.attrs["gtype"] = str(GeometryType.POINT)
+
+    coords = {"x": x, "y": y, "z": z}
+    for coord, value in coords.items():
+        if value is not None:
```

Comment:
What if neither of these are true? e.g. if value=None - would we not want to set to np.nan? 

Would it make sense to:

    ds.coords[coord] = value if value is not None else np.nan

?